lib/utils/export/to-zip.js

```diff
@@ -0,0 +1,21 @@
+import JSZip from 'jszip';
+
+export const noteExportToZip = notes => {
+	const zip = new JSZip();
+
+	notes
+		.activeNotes
+		.forEach(
+			( { id, content } ) => zip.file( `${ id }.txt`, content )
```

Comment:
One of the things users complain about with the current zip export is that it uses the id as the file name. If there's a way that we can use the note's first line, that'd be cool. What we need to watch out for is notes with the same first line so we don't overwrite the file!